code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -221,6 +221,7 @@
 	see_in_dark = 8
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
 	var/independent = FALSE
+	var/list/blobs_in_area
```

Comment:
w h y

Reply:
it was either this or calling range 3 times per life tick

Reply:
don't put this list as a var on the mob jesus christ

Reply:
ok